src/pages/SearchPage.js

```diff
@@ -164,36 +162,32 @@ class SearchPage extends Component {
 
     render() {
         const sections = this.getSections();
-        const headerMessage = getHeaderMessage(
-            (this.state.recentReports.length + this.state.personalDetails.length) !== 0,
-            Boolean(this.state.userToInvite),
-            this.state.searchValue,
-        );
         return (
             <ScreenWrapper>
                 {({didScreenTransitionEnd}) => (
-                    <>
-                        <HeaderWithCloseButton
-                            title={this.props.translate('common.search')}
-                            onCloseButtonPress={() => Navigation.dismissModal(true)}
-                        />
-                        <View style={[styles.flex1, styles.w100, styles.pRelative]}>
-                            <FullScreenLoadingIndicator visible={!didScreenTransitionEnd} />
-                            {didScreenTransitionEnd && (
-                            <OptionsSelector
-                                sections={sections}
-                                value={this.state.searchValue}
-                                onSelectRow={this.selectReport}
-                                onChangeText={this.onChangeText}
-                                headerMessage={headerMessage}
-                                hideSectionHeaders
-                                hideAdditionalOptionStates
-                                showTitleTooltip
+                    didScreenTransitionEnd && (
+                        <>
+                            <HeaderWithCloseButton
+                                title={this.props.translate('common.search')}
+                                onCloseButtonPress={() => Navigation.dismissModal(true)}
                             />
-                            )}
-                        </View>
-                        <KeyboardSpacer />
-                    </>
+                            <View style={[styles.flex1, styles.w100, styles.pRelative]}>
+                                <FullScreenLoadingIndicator visible={!didScreenTransitionEnd} />
+                                {didScreenTransitionEnd && (
+                                <OptionsSelector
+                                    sections={sections}
+                                    onSelectRow={this.selectReport}
+                                    searchSections={this.searchSections}
```

Comment:
This prop needs to be passed to other pages where this component is used. Otherwise, Search will not work there.